bokeh/server/views/backbone.py

```diff
@@ -286,6 +288,8 @@ def update(docid, typename, id):
 @handle_auth_error
 def delete(docid, typename, id):
     #I don't think this works right now
+    obj = obj  # noqa - fool pyflakes - bug or magic?
+    clientdoc = clientdoc  # noqa
     doc = docs.Doc.load(bokeh_app.servermodel_storage, docid)
```

Comment:
Someone needs to have a close look at this. I cannot see how `clientdoc` and `obj` would be available in this namespace. Maybe some black magic in the decorator (at first glance it does't seem so)?. This pattern occurs several times in this module.